gui/src/components/Taipy/Chart.tsx

```diff
@@ -355,9 +356,30 @@ const Chart = (props: ChartProp) => {
     }, [active, props.plotConfig]);
 
     const onRelayout = useCallback(
-        (eventData: PlotRelayoutEvent) =>
-            tp_onRangeChange && dispatch(createSendActionNameAction(id, { action: tp_onRangeChange, ...eventData })),
```

Comment:
Should we not keep the on_range_change for other uses (when there's no decimator) ?